Tests/LibCompress/TestZlib.cpp

```diff
@@ -48,29 +46,22 @@ TEST_CASE(zlib_decompress_stream)
 
 TEST_CASE(zlib_compress_simple)
 {
-    // Note: This is just the output of our compression function from an arbitrary point in time.
-    // This test is intended to ensure that the decompression doesn't change unintentionally,
-    // it does not make any guarantees for correctness.
-
-    Array<u8, 37> const compressed {
-        0x78, 0x9C, 0x0B, 0xC9, 0xC8, 0x2C, 0x56, 0xC8, 0x2C, 0x56, 0x48, 0x54,
-        0x28, 0xCE, 0xCC, 0x2D, 0xC8, 0x49, 0x55, 0x28, 0x49, 0xAD, 0x28, 0x51,
-        0x48, 0xCB, 0xCC, 0x49, 0x55, 0xB0, 0xD2, 0x04, 0x00, 0x99, 0x5E, 0x09,
-        0xE8
+    Array<u8, 34> const compressed {
+        0x78, 0x9C, 0x0B, 0xC9, 0xC8, 0x2C, 0x56, 0x00, 0xA2, 0x44, 0x85, 0xE2,
+        0xCC, 0xDC, 0x82, 0x9C, 0x54, 0x85, 0x92, 0xD4, 0x8A, 0x12, 0x85, 0xB4,
+        0x4C, 0x20, 0xCB, 0x4A, 0x13, 0x00, 0x99, 0x5E, 0x09, 0xE8
```

Comment:
In the other test case you've removed the non-deterministic results; are these deterministic? And if not, why remove the comment?